701 - Fix expanded panes with accordion (#3316)
infor-design/enterprise: docs/CHANGELOG.md

- `[About]` Fixed a bug where the browser language was shown as the locale name, we now show browser language and IDs language and locale separate. ([#2913](https://github.com/infor-design/enterprise/issues/2913))
- `[About]` Fixed a bug where the OS version was duplicated. ([#1650](https://github.com/infor-design/enterprise/issues/1650))
- `[Accordion]` Fixed inconsistency style of focus element after clicking on a certain accordion header. ([#3082](https://github.com/infor-design/enterprise/issues/3082))

> - `[Accordion]` Fixed an issue where all panes were expanded and there was no way to be close. ([#701](https://github.com/infor-design/enterprise-ng/issues/3217))

Reviewer: ```suggestion
- `[Accordion]` Fixed an issue that when all panes are expanded then they could no longer be closed. ([#701](https://github.com/infor-design/enterprise-ng/issues/3217))
```
